rerun-failed-run/README.md

```diff
@@ -0,0 +1,66 @@
+# rerun-failed-run
+
+This reusable workflow can be used to retrigger in GitHub Actions workflows to automatically retrigger them if they failed and contain a specific error message in their build logs.
+
+## Usage
+
+The workflow can be invoked with the aid of the GitHub CLI from another workflow. In case the calling workflow does not belong in the same repository the designated GitHub App Token should be generated in order to allow the invocation. For that purpose, a github app is already set up, and its credentials can be found in the following Vault secret paths:
```

Comment:
```suggestion
The workflow can be invoked with the aid of the GitHub CLI from another workflow. In case the calling workflow is not in the same repository, the designated GitHub App Token should be generated to allow the invocation. For that purpose, a GitHub app is already set up, and its credentials can be found in the following Vault secret paths:
```